OpenRA.Game/Settings.cs

```diff
@@ -90,18 +90,32 @@ public class DebugSettings
 		public bool SanityCheckUnsyncedCode = false;
 		public int Samples = 25;
 		public bool IgnoreVersionMismatch = false;
+
+		[Desc("Dump performance data into cpu.csv and render.csv accessible via ~/.openra/Logs.")]
+		public bool Benchmark = false;
 	}
 
 	public class GraphicSettings
 	{
 		public string Renderer = "Default";
+
+		[Desc("This can be set to Windowed, Fullscreen or PseudoFullscreen.")]
 		public WindowMode Mode = WindowMode.PseudoFullscreen;
+
+		[Desc("Screen resolution in fullscreen mode.")]
 		public int2 FullscreenSize = new int2(0, 0);
+
+		[Desc("Screen resolution in windowed mode.")]
 		public int2 WindowedSize = new int2(1024, 768);
+
 		public bool HardwareCursors = true;
 		public bool PixelDouble = false;
```

Comment:
For the number of actors in an average screen you'll be halving a negligible number to make another negligible number.  A lot of the rendering cost is near constant (ui, terrain/resources/shroud, syncing with the gpu, etc) given the small number of vertices we are actually pushing.